test/HttpServer.test.ts

```diff
@@ -138,4 +144,41 @@ describe('HttpServer', () => {
             );
         });
     });
+
+    describe('RelayHandler', () => {
+        const fakeResponseExpress: SinonStubbedInstance<Partial<Response>> = {
+            send: stub()
+        };
+        let fakeRequestExpress: SinonStubbedInstance<Partial<Request>>;
+
+        it('should return a reponse with signedTx and transactionHash in body', async () => {
+            const fakeResponseRelayTransaction: SignedTransactionDetails = {
+                signedTx:
+                    '0xf9036a1b8403e252e08301f9699466fa9feafb8db66fe2160ca7aeac7fc24e25438780b90304180e59260000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000028000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000003e252e0a81483953da7601ef828906dbab2e4baf21ddfd3d3c484fe7c43c55836c6c77200000000000000000000000074105590d404df3f384a099c2e55135281ca6b40000000000000000000000000eab5b9fa91aeffaa9c33f9b33d12ab7088fa7f6f0000000000000000000000005c6e96a84271ac19974c3e99d6c4be4318bfe48300000000000000000000000066fa9feafb8db66fe2160ca7aeac7fc24e254387000000000000000000000000cb8f8130e289946aec9a17b29819716b1e9e49980000000000000000000000000000000000000000000000000000000000000000000000000000000000000000f5859303f76596dd558b438b18d0ce0e1660f3ea0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000050000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000160000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000411285a1fa3217b0b8ca9b23ee2404324c965af9bb3908199ffc8bc7a47f59cef9160a142da5269fa5b7bfa8a688c1a507bedeba0650f1d617b93c8ece598aba651c0000000000000000000000000000000000000000000000000000000000000062a053917eb36fd417620eb21f6fa35b701a9efa2dc4a39422c04e1469b21b1c0b63a010ff3e24bc8421bb801abb914c3f6031ecf99e7bb8d514eae77bbb7db8d1208c',
+                transactionHash:
+                    '0xb8c646c863ff648b6f75f05cbcd84625521ca802d397e6473ba8f5e00e65f169'
+            };
+
+            fakeRelayServer.createRelayTransaction = (
+                req: RelayTransactionRequest | DeployTransactionRequest
+            ) => Promise.resolve(fakeResponseRelayTransaction);
+            fakeRequestExpress = {
+                body: {}
+            };
+
+            await httpServer.relayHandler(
+                fakeRequestExpress as Request,
+                fakeResponseExpress as Response
+            );
+
+            assert.containsAllKeys(fakeResponseExpress.send?.args[0][0], [
```

Comment:
There should be a method to check exactly the arguments which a method call has been called with (e.g.: `to.be.called.withExactly`)